gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/anonymous/AnonymousConfig.groovy

```diff
@@ -31,18 +34,32 @@ class AnonymousConfig extends WebSecurityConfigurerAdapter {
   static String key = "spinnaker-anonymous"
   static String defaultEmail = "anonymous"
 
+  @Autowired
+  AccountsService accountsService
+
+  List<String> anonymousAllowedAccounts = []
```

Comment:
I'd make this a concurrent collection (CopyOnWriteArrayList?) since it is getting mutated in a background thread